data/lib/src/network/model/request/add_upload_body_request.dart

```diff
@@ -75,7 +77,24 @@ class AddUploadBodyRequest with EquatableMixin {
 
   factory AddUploadBodyRequest.fromJson(Map<String, dynamic> json) => _$AddUploadBodyRequestFromJson(json);
 
-  Map<String, dynamic> toJson() => _$AddUploadBodyRequestToJson(this);
```

Comment:
What problem with it?

Reply:
The generation code causes an error when parsing the request object